src/common/utils.ts

```diff
@@ -182,7 +182,7 @@ export const throttle = (func: (...args) => void, wait: number): (...args) => vo
  * @param data {Object} - Data to be added
  *
  * @example
- * StringFormat(this will log {time}`, { time: '1234' })
+ * StringFormat('this will log {time}', [{ time: '1234' }])
```

Comment:
@vrbsm The 2nd arg should be `object`. As you can see we are using `for..in` loop

Reply:
We need to fix the `formatString` method 🙂